crates/pera-types/src/collection_types.rs

```diff
@@ -93,6 +93,13 @@ pub struct Bag {
     pub size: u64,
 }
 
+/// Rust version of the Move pera::bag::Bag type.
+#[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
```

Comment:
Was this generated or did you add it manually?

Reply:
added manually